hazelcast/include/hazelcast/client/HazelcastClient.h

```diff
@@ -713,6 +729,14 @@ namespace hazelcast {
             internal::nearcache::NearCacheManager &getNearCacheManager();
 
             serialization::pimpl::SerializationService &getSerializationService();
+
```

Comment:
We should not expose these methods to user ? 
getNearCacheManager and getSerializationService seems already exposed. We should either remove or put documentation here for users to not use them.

Reply:
I separated the client impl and the user API.